feeluown/player.py

```diff
@@ -137,7 +137,7 @@ async def a_set_current_song(self, song):
             else:
                 logger.info('find song standby failed: not found')
                 final_song = song
-            self._set_current_song(final_song, final_song.url)
+            self.current_song = final_song
```

Comment:
The current fix should work, however, this may cause potential infinitely-standby-fetching.

Currently, only `Media` object contains bitrate info, the player can't parse bitrate from `song.url`. I think the root cause is that the `library.a_list_song_standby(song)` method only return a song object. It should return more info (including a `media` object).

I will try to refactor the `a_list_song_standby` method lator.

Reply:
@BruceZhang1993 suggests that the old method will not follow audio_select_policy (like fetching shq quality media), i think following the policy is also preferred whenever it doesn't cause an infinite loop.